content/arduino-cloud/09.business/01.iso27001/iso27001.md

```diff
@@ -10,15 +10,16 @@ author: 'Arduino Security Team'
 ***For any questions regarding security, contact the [Arduino Security Team](mailto:security@arduino.cc).***  
 
 
-![Arduino Cloud services are officially certified for ISO/IEC 27001:2013 (ISO 27001)](./assets/infosec.png)
+![Arduino Cloud services are officially certified for ISO/IEC 27001:2012 (ISO 27001)](./assets/infosec.png)
```

Comment:
Looks like is wrong,

Should be:

"also the description of the picture is now wrong because we need to refer everywhere to ISO27001:2022 instead of 2013"

please re-check chat and make sure we replace of the references with the new one!